examples/dynamoathenamigration/migration.go

```diff
@@ -556,6 +565,12 @@ func (t *task) emitEvents(ctx context.Context, eventsC <-chan apievents.AuditEve
 		if count == 0 {
 			return errors.New("there were not events from export")
 		}
+		if len(invalidEvents) > 0 {
+			for _, invalidE := range invalidEvents {
+				t.Logger.Debugf("Event %s %s %s is invalid", invalidE.GetType(), invalidE.GetID(), invalidE.GetTime().Format(time.RFC3339))
```

Comment:
Especially because we might have empty strings or other weird things going on:
```suggestion
				t.Logger.Debugf("Event %q %q %v is invalid", invalidE.GetType(), invalidE.GetID(), invalidE.GetTime().Format(time.RFC3339))
```